language/move-analyzer/editors/code/src/context.ts

```diff
@@ -10,10 +10,15 @@ import { sync as commandExistsSync } from 'command-exists';
 
 /** Information passed along to each VS Code command defined by this extension. */
 export class Context {
+    private _client: lc.LanguageClient | undefined;
```

Comment:
unless required by the style guide in this repo, it's more idiomatic to rely on the `private` declaration without the `_` prefix in the name